transactions/set-version-boundary/2025/jun-3/README.md

```diff
@@ -0,0 +1,18 @@
+# Set version boundary
+> June 3rd, 2025 Height Coordinated Upgrade (HCU)
+
+This transaction sets the height at which the height coordinated upgrade occurs on mainnet.
+
+Version to set: `v0.42.0-rc.2`
```

Comment:
Are we performing an HCU transaction at the same time?

Reply:
yes